src/main/java/org/fundacionjala/coding/Main.java

```diff
@@ -0,0 +1,20 @@
+package org.fundacionjala.coding;
+
+public class Main {
+    public static void main(String[] args) {
+        Customer customer;
+        Movie movie;
+        Rental rental;
+        customer = new Customer("Test");
+
+        movie = new MovieRegular("Terminator");
+        rental = new Rental(movie,2);
+        customer.addRental(rental);
+
+        movie = new MovieNew("The Revenant");
+        rental = new Rental(movie,2);
+        customer.addRental(rental);
+
+        System.out.println(customer.statement());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)